react/features/prejoin/reducer.js

```diff
@@ -28,7 +31,16 @@ const DEFAULT_STATE = {
     videoTrack: null,
     audioTrack: null,
     contentSharingTrack: null,
-    rawError: ''
```

Comment:
This is an example where we failed to keep alpha ordering. But now that you're here and modifying this object quite heavily, we'd be super happy if you fixed this ;)

Reply:
Fixed.